pyuvdata/miriad.py

```diff
@@ -410,8 +425,8 @@ def read_miriad(self, filepath, correct_lat_lon=True, run_check=True, run_check_
             warnings.warn(str(ve))
 
         # check if object has all required uv_properties set
-        if run_check:
-            self.check(run_check_acceptability=run_check_acceptability)
+#        if run_check:
+#            self.check(run_check_acceptability=run_check_acceptability)
```

Comment:
Are you sure you want the check lines to be commented out? That seems like a bad idea.